package.json

```diff
@@ -5,5 +5,6 @@
   "main": "index.js",
   "repository": "https://github.com/pesto-students/batch-10-tripen.git",
   "license": "MIT",
+  "scripts": {},
```

Comment:
why this is empty?

Reply:
Will update scripts with commands for running client & server